package.json

```diff
@@ -31,17 +31,17 @@
   },
   "private": false,
   "dependencies": {
-    "@angular/animations": "~10.1.0",
-    "@angular/common": "~10.1.0",
-    "@angular/compiler": "~10.1.0",
-    "@angular/core": "~10.1.0",
-    "@angular/forms": "~10.1.0",
-    "@angular/localize": "^10.1.0",
```

Comment:
Maybe it's enough just to change from ^ to ~. There is no need to upgrade versions for all libs.

Reply:
It is enough, but maybe we should bump it to the most recent version that won't break our build?

Reply:
It also won't break if you put ti just with ~, because our app is supposed to work also with 10.1 version of Angular. And other people maybe have exactly that local version.